librz/core/cbin.c

```diff
@@ -2916,6 +2918,12 @@ static int bin_sections(RzCore *r, PJ *pj, int mode, ut64 laddr, int va, ut64 at
 		if (plugin_flags_support && !print_segments) {
 			rz_table_set_columnsf(table, "s", "flags");
 		}
+		if (is_elf && print_segments) {
+			rz_table_set_columnsf(table, "x", "align");
+		}
+		if (is_mach0 && !print_segments) {
+			rz_table_set_columnsf(table, "x", "align");
+		}
```

Comment:
If you think using something like `print_align` to avoid duplicate line is better, I can make changes.

Reply:
Good idea, please do.